costmap:
   global_frame: map
   robot_base_frame: base_link
   map_topic: map

   always_send_full_costmap: true
   static_map: false
   
   rolling_window: false
   
   update_frequency: 10
   publish_frequency: 10
   
   transform_tolerance: 1
   resolution: 0.05
   robot_radius: 0.30 
      
   inflation_layer: 
      cost_scaling_factor: 1
      enabled: true
      inflate_unknown: false
      inflation_radius: 1
      
   static_layer:
      unknown_cost_value: -1
      lethal_cost_threshold: 100

   obstacles:
      observation_sources: scan
      enabled: true
      scan: {data_type: LaserScan, sensor_frame: /base_link, clearing: true, marking: true, topic: /scan}
      inflation_radius: 0

   plugins:
      - {name: static_layer, type: "costmap_2d::StaticLayer"}
      - {name: obstacles, type: "costmap_2d::VoxelLayer"}
      - {name: inflation_layer, type: 'costmap_2d::InflationLayer'}
      